secretless-k8s-tutorial/intro.md

```diff
@@ -0,0 +1,22 @@
+<p align="center">
+  <img src="assets/cyberark_logo.png">
+</p>
+
+# Introduction
+
+In this scenario, you will get an overview of the Secretless Broker, and you will learn how to deploy Secretless Broker, along with an application that has *no knowledge of passwords* and a database backend.
```

Comment:
Nit: 
"...you will get an overview of the Secretless Broker and learn how to deploy it, along with an application..." is a little more concise and readable, in my opinion.